### PR TITLE
Enable offline google analytics

### PR DESCRIPTION
**Pull request by deepaklalwani97**

### CODE REVIEW

File: inc/classes/class-service-worker.php [hunk 1]
```diff
@@ -95,4 +97,24 @@ public function cache_images( \WP_Service_Worker_Scripts $scripts ) {
...
 	}
 
+	/**
+	 * Enables offline google analytics.
+	 *
+	 * @param object $scripts
+	 *
+	 * @return void
+	 */
+	public function enable_offline_google_analytics( $scripts ) {
```
Comment (chandrapatel) [inline inc/classes/class-service-worker.php]: `\WP_Service_Worker_Scripts $scripts` in param.

File: inc/classes/class-service-worker.php [hunk 2]
```diff
@@ -95,4 +97,24 @@ public function cache_images( \WP_Service_Worker_Scripts $scripts ) {
 
 	}
 
+	/**
+	 * Enables offline google analytics.
+	 *
+	 * @param object $scripts
```
Comment (chandrapatel) [inline inc/classes/class-service-worker.php]: `@param \WP_Service_Worker_Scripts $scripts Instance to register service worker behavior with.`

**Review by pradeep910 — DISMISSED** [review]

File: inc/classes/class-service-worker.php (hunk 2 above)
Comment (pradeep910) [reply inc/classes/class-service-worker.php]: @deepaklalwani97 We can merge after this change.

**Comment by deepaklalwani97** [comment]
@pradeep910 @chandrapatel Addressed feedbacks need your approval again to merge this PR.

**Review by chandrapatel — APPROVED** [review]

**Pull request merged**